skrub/tests/test_on_subframe.py

```diff
@@ -70,7 +71,16 @@ def test_empty_output(df_module, use_fit_transform):
         out = transformer.fit_transform(df)
     else:
         out = transformer.fit(df).transform(df)
-    df_module.assert_frame_equal(out, s.select(df, ()))
+    selected = s.select(df, ())
```

Comment:
It would be useful to add a comment explaining what the above line is doing (I cannot tell)